examples/batch_solve.rs

```diff
@@ -1,27 +1,86 @@
 use std::{
+    collections::HashMap,
+    fs::{remove_file, File},
+    io::BufReader,
     path::{Path, PathBuf},
-    process::exit,
 };
 
 use clap::Parser;
 use postflop_solver::{
-    cards_from_str, deserialize_configs_from_file, save_data_to_file, serialize_configs_to_json,
-    solve, ActionTree, BoardState, PostFlopGame, Range,
+    card_to_string, cards_from_str, flop_from_str, save_data_to_file, solve, Action, ActionTree,
+    BoardState, CardConfig, PostFlopGame, Range, TreeConfig,
 };
+use serde::{de::DeserializeOwned, Deserialize, Serialize};
+
+const METADATA_FILENAME: &str = "meta.sdb";
```

Comment:
Maybe just a `.json` extension?

```suggestion
const METADATA_FILENAME: &str = "meta.json";
```